recipes/progress_reporter/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "progress_reporter" %}
+{% set version = "1.2" %}
+
+package:
+  name: {{ name|lower }} 
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/marscher/progress_reporter
+  git_rev: {{ 'v' + version }}
+
+build:
+  noarch_python: True
+  number: 0
+# since this is noarch, only build on linux
+  skip: True  # [not linux]
```

Comment:
We only need to build it once, since it is a no arch package. I could also build it on windoze and skip all the other platforms, because we need only one package.